src/app/_core/constants/pagination.constants.ts

```diff
@@ -1,4 +1,4 @@
-export const limitParam = 12;
-export const pageParam = 0;
-export const pokemonNumber = 1025;
-export const pageSizeOptions = [12, 24, 48, 72];
+export const limitParam: number = 12;
+export const pageParam: number = 0;
+export const pokemonNumber: number = 1025;
+export const pageSizeOptions: number[] = [12, 24, 48, 72];
```

Comment:
Usually it's a good practice to put constant in capital:

LIMIT_PAGE = 12;
DEFAULT_PAGE = 0;

This way it's easier to understand that they are constant when you read the code some other place